packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -15,6 +15,85 @@ import { Dictionary } from '../typedefs';
 
 const STARTING_Z_INDEX = 2147400000;
 
+export interface EnqueueLinksByClickingElementsOptions {
+    /**
+     *   Puppeteer [`Page`](https://pptr.dev/#?product=Puppeteer&show=api-class-page) object.
```

Comment:
lets remove the redundant spacing (general note for the whole PR)

```suggestion
     * Puppeteer [`Page`](https://pptr.dev/#?product=Puppeteer&show=api-class-page) object.
```

Reply:
Hopefully did not miss it anywhere, removed in files in current PR